src/deployconfig.jl

```diff
@@ -489,20 +489,21 @@ function post_status(::GitHubActions; type, repo::String, subfolder = nothing, k
             sha = get(ENV, "GITHUB_SHA", nothing)
         end
         sha === nothing && return
-        return post_github_status(type, repo, sha, subfolder)
+        return post_github_status(type, gha.github_repository, repo, sha, subfolder)
     catch
         @debug "Failed to post status"
     end
 end
 
-function post_github_status(type::S, deploydocs_repo::S, sha::S, subfolder = nothing) where {S <: String}
+function post_github_status(type::S, source::S, deploydocs_repo::S, sha::S, subfolder = nothing) where {S <: String}
     try
         Sys.which("curl") === nothing && return
-        ## Extract owner and repository name
+        ## Extract owner and repository names
+        source_owner, source_repo = split(source, '/')
```

Comment:
Is there any chance `gha.github_repository` includes the `.git` extension?  In that case I'd need to drop it with a regex like done below.

Reply:
I think it's fine to assume no `.git`.